src/main/java/com/akolb/HMSClient.java

```diff
@@ -206,12 +206,42 @@ void createDatabase(@NotNull String name) throws TException {
     client.create_database(db);
   }
 
+  /**
+   * Create database but convert any exception to unchecked {@link RuntimeException}
```

Comment:
Nit - according to Javadoc conventions, the first sentence should end with dot and shouldn't include links - other sentences can. I am not following this consistently myself here (I should), let's not make it worse.